src/module.c

```diff
@@ -222,8 +242,14 @@ int TSDB_info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
         series = RedisModule_ModuleTypeGetValue(key);
     }
 
-    RedisModule_ReplyWithArray(ctx, 7*2);
+    RedisModule_ReplyWithArray(ctx, 10*2);
 
+    RedisModule_ReplyWithSimpleString(ctx, "totalSamples");
```

Comment:
@K-Jo do we want to return `totalSamples` on  every info requests? there's a tradeoff here, calculating it is O(n) which can take some time.
How do you see people using it?

Reply:
@K-Jo @danni-m 
Same goes for memory usage